edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -173,9 +173,26 @@ def _set_filters(token):
 
 def _verify_jwt_signature(token, jwt_issuer, decode_symmetric_token):
```

Comment:
On master, Robert has made some changes to this method that duplicate the `JMS().verify_compact` call to add instrumentation around symmetric and asymmetric key usage. I believe he had intended to have that instrumentation removed by the time this PR was ready, but this one might be ready first.

My recommendation would be to move the for-block into its own private method (maybe `_verify_jwt_signature_inner`? no strong preference) or even just a closure so that that single call can be duplicated. Then you'll be able to merge or rebase to pick up the new changes.

Reply:
Please also remember to bump the version and add a changelog entry.